chia/util/keychain.py

```diff
@@ -203,6 +210,21 @@ def mnemonic_to_seed(mnemonic: str, passphrase: str) -> bytes:
     return seed
 
 
+def default_keychain_user() -> str:
+    return DEFAULT_USER
+
+
+def default_keychain_service() -> str:
+    return DEFAULT_SERVICE
```

Comment:
is there a reason to have these functions, rather than just using the constants instead?

Reply:
I can revisit this later if needed.